### PR TITLE
feat(components): Update batch prediction component

### PR DESCRIPTION
**Pull request by IronPan**
**Description of your changes:**
/assign @SinaChavoshi 

**Checklist:**
- [x] The title for your pull request (PR) should follow our title convention. [Learn more about the pull request title convention used in this repository](https://github.com/kubeflow/pipelines/blob/master/CONTRIBUTING.md#pull-request-title-convention).

### CODE REVIEW

File: components/google-cloud/google_cloud_pipeline_components/container/experimental/gcp_launcher/batch_prediction_job_remote_runner.py [hunk 1]
```diff
@@ -14,9 +14,17 @@
...
+
+ARTIFACT_PROPERTY_KEY_BIGQUERY_OUTPUT_TABLE = 'bigqueryOutputTable'
+ARTIFACT_PROPERTY_KEY_BIGQUERY_OUTPUT_DATASET = 'bigqueryOutputDataset'
+ARTIFACT_PROPERTY_KEY_GCS_OUTPUT_DIRECTORY = 'gcsOutputDirectory'
 
 
 def create_batch_prediction_job_with_client(job_client, parent, job_spec):
+    # todo remove
+    print('**')
+    print(job_spec)
```
Comment (SinaChavoshi) [inline components/google-cloud/google_cloud_pipeline_components/container/experimental/gcp_launcher/batch_prediction_job_remote_runner.py:27]: nit: Seems like testing artifacts, should be removed?

File: components/google-cloud/google_cloud_pipeline_components/container/experimental/gcp_launcher/batch_prediction_job_remote_runner.py [hunk 2]
```diff
@@ -61,6 +70,17 @@ def create_batch_prediction_job(
             create_batch_prediction_job_with_client, payload)
 
     # Poll batch prediction job status until "JobState.JOB_STATE_SUCCEEDED"
-    remote_runner.poll_job(get_batch_prediction_job_with_client, job_name)
+    get_job_response= remote_runner.poll_job(get_batch_prediction_job_with_client, job_name)
+    # todo remove
+    print(get_job_response)
```
Comment (SinaChavoshi) [inline components/google-cloud/google_cloud_pipeline_components/container/experimental/gcp_launcher/batch_prediction_job_remote_runner.py:75]: nit: testing artifact should be removed.

**Review by SinaChavoshi — APPROVED** [review]
Thank you for PR.

File: components/google-cloud/google_cloud_pipeline_components/container/experimental/gcp_launcher/batch_prediction_job_remote_runner.py (hunk 1 above)
Comment (IronPan) [reply components/google-cloud/google_cloud_pipeline_components/container/experimental/gcp_launcher/batch_prediction_job_remote_runner.py]: done

File: components/google-cloud/google_cloud_pipeline_components/container/experimental/gcp_launcher/batch_prediction_job_remote_runner.py (hunk 2 above)
Comment (IronPan) [reply components/google-cloud/google_cloud_pipeline_components/container/experimental/gcp_launcher/batch_prediction_job_remote_runner.py]: done

**Pull request merged**